CONTRIBUTING.md

```diff
@@ -1 +1,10 @@
-# TODO
+# Contribution Guidelines
+
+## TODO
+
+## Pull Requests
+
+* Include one feature/bug fix per Pull Request.  Multiple issues in one PR can increase the complexity of the review and could delay the merging of your code.
```

Comment:
It might also be good to add this info to https://github.com/arnesson/cordova-plugin-firebase/blob/master/PULL_REQUEST_TEMPLATE.md

Reply:
Updated the branch with a new commit to include this

Reply:
Awesome.  Feel free to merge the PR yourself once you feel you're done with the doc updates (you should have authority to merge PRs, if not let me know).